cmake/modules/RootBuildOptions.cmake

```diff
@@ -161,6 +161,7 @@ ROOT_BUILD_OPTION(tmva-pymva ON "Enable support for Python in TMVA (requires num
 ROOT_BUILD_OPTION(tmva-rmva OFF "Enable support for R in TMVA")
 ROOT_BUILD_OPTION(unuran OFF "Enable support for UNURAN (package for generating non-uniform random numbers)")
 ROOT_BUILD_OPTION(vc OFF "Enable support for Vc (SIMD Vector Classes for C++)")
+ROOT_BUILD_OPTION(vmc ON "Build VMC simulation library")
```

Comment:
If the idea is to later deprecate this and remove, maybe the default value for this option should be `OFF` rather than `ON`, no?